examples/inference-deployments/mpt/mpt_7b_simple.yaml

```diff
@@ -0,0 +1,12 @@
+name: mpt-7b-simple
+compute:
```

Comment:
Can we also not specify this for no-code?

Reply:
nah still need to specify what resources you want